### PR TITLE
fix: update og:image with optimized version

### LINKED ISSUE #57: Support opengraph tags
#### Reason/Context

I shared the link on Twitter, and noticed we didn't have any open graph tags.

Be great to get some open graph tags for the conference for nice media sharing on things like Slack, Twitter etc.

### PR DESCRIPTION
**Pull request by mcturco**
**Description**

- Optimized og:image for web

**Related issue(s)**
Resolves #57

### CODE REVIEW

File: sources/pug/_config.pug [hunk 1]
```diff
@@ -94,4 +94,13 @@
...
+    pageName = 'Home',
+
+    // Meta tags
+    ogData = {
+      title: 'AsyncAPI Online Conference',
+      image: '/images/meta/og-image.jpg',
+      description: 'Join us at the AsyncAPI Online Conference and learn about Open-Source and Event-Driven Architectures.',
+      twitterCard: 'summary_large_image',
+      twitterUrl: 'https://twitter.com/asyncapispec'
+    }
```
Comment (magicmatatjahu) [inline sources/pug/_config.pug:106]: In slack it works 👏🏼  But I don't know why, but there are still that the same changes as in master repo 🤔  Also as I see you have conflicts, so you should fix them, and then that changes should dissapear :)

image


image

Comment (derberg) [reply sources/pug/_config.pug:106]: @mcturco I think you added new image version to old branch you already had, but it was merged already, thus conflicts, you will have to open new branch from latest master in your fork and then new PR

Comment (mcturco) [reply sources/pug/_config.pug:106]: Oh shoot okay, my bad

Comment (mcturco) [reply sources/pug/_config.pug:106]: @magicmatatjahu @derberg opened a new PR here: https://github.com/asyncapi/conference-website/pull/73 sorry about that guys!

Comment (derberg) [reply sources/pug/_config.pug:106]: @mcturco no worries!

**Pull request closed without merging**